packages/typespec-client-generator-core/test/decorators.test.ts

```diff
@@ -3306,7 +3312,8 @@ describe("typespec-client-generator-core: decorators", () => {
       strictEqual(clientAccessor.name, "getInterfaceV2");
```

Comment:
should we add linter rule for the name if it's going to be a subclient of a client? Another question is since we are using interface name as the sub client name, would it be confusing to add Client suffix if we do add that linter rule? 
Also, how can we tell if we want this interface to be a subclient of the current client of just an operation group of the current client?

Reply:
what's lint rule? i don't quite get you. this name is the method name, not the client name. there is another issue to track adding `groupPath` in legacy `SdkOperationGroup` type for the new interface. And operation group in the new interface is also a client, but with private initialization.

Reply:
`getInterfaceV2` isn't the name of the subclient, it's the name of the client accessor method. We have linter rules when people use the `@client` decorator